[2025-08-18T13:23:19.337+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-18T13:23:19.410+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: youtube_pipeline_batch_data.run_q_03_gold_combinations manual__2025-08-18T13:16:17.589887+00:00 [queued]>
[2025-08-18T13:23:19.430+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: youtube_pipeline_batch_data.run_q_03_gold_combinations manual__2025-08-18T13:16:17.589887+00:00 [queued]>
[2025-08-18T13:23:19.431+0000] {taskinstance.py:2303} INFO - Starting attempt 3 of 4
[2025-08-18T13:23:19.458+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): run_q_03_gold_combinations> on 2025-08-18 13:16:17.589887+00:00
[2025-08-18T13:23:19.486+0000] {standard_task_runner.py:63} INFO - Started process 4947 to run task
[2025-08-18T13:23:19.500+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'youtube_pipeline_batch_data', 'run_q_03_gold_combinations', 'manual__2025-08-18T13:16:17.589887+00:00', '--job-id', '306', '--raw', '--subdir', 'DAGS_FOLDER/youtube_pipeline.py', '--cfg-path', '/tmp/tmpvzy2vfzx']
[2025-08-18T13:23:19.506+0000] {standard_task_runner.py:91} INFO - Job 306: Subtask run_q_03_gold_combinations
[2025-08-18T13:23:19.664+0000] {task_command.py:426} INFO - Running <TaskInstance: youtube_pipeline_batch_data.run_q_03_gold_combinations manual__2025-08-18T13:16:17.589887+00:00 [running]> on host 47205959681c
[2025-08-18T13:23:19.952+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='youtube_pipeline_batch_data' AIRFLOW_CTX_TASK_ID='run_q_03_gold_combinations' AIRFLOW_CTX_EXECUTION_DATE='2025-08-18T13:16:17.589887+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-18T13:16:17.589887+00:00'
[2025-08-18T13:23:19.953+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-18T13:23:20.004+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-08-18T13:23:20.005+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'PGPASSWORD=citus psql -h citus_coordinator -U citus -d youtube_dw -f /opt/***/sql/q_03_gold_combinations']
[2025-08-18T13:23:20.028+0000] {subprocess.py:86} INFO - Output:
[2025-08-18T13:23:20.218+0000] {subprocess.py:93} INFO - psql: error: /opt/***/sql/q_03_gold_combinations: No such file or directory
[2025-08-18T13:23:20.220+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2025-08-18T13:23:20.222+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-18T13:23:20.259+0000] {taskinstance.py:2890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/bash.py", line 243, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2025-08-18T13:23:20.270+0000] {taskinstance.py:1205} INFO - Marking task as UP_FOR_RETRY. dag_id=youtube_pipeline_batch_data, task_id=run_q_03_gold_combinations, execution_date=20250818T131617, start_date=20250818T132319, end_date=20250818T132320
[2025-08-18T13:23:20.329+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 306 for task run_q_03_gold_combinations (Bash command failed. The command returned a non-zero exit code 1.; 4947)
[2025-08-18T13:23:20.364+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-08-18T13:23:20.429+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-08-18T13:23:20.438+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
